### PR TITLE
Fix for including nvtx dll and cudart

### PR DESCRIPTION
**Pull request by atalman**
Fix for including nvtx dll and cudart.

Test: https://github.com/pytorch/pytorch/actions/runs/3177623914

### CODE REVIEW

File: conda/pytorch-nightly/bld.bat [hunk 1]
```diff
@@ -120,6 +117,8 @@ if NOT "%build_with_cuda%" == "" (
     if exist "C:\Windows\System32\zlibwapi.dll" (
         copy "C:\Windows\System32\zlibwapi.dll"  %SP_DIR%\torch\lib
     )
+
```
Comment (izaitsevfb) [inline conda/pytorch-nightly/bld.bat:120]: nit: extra line breaks

**Review by izaitsevfb — APPROVED** [review]
LGTM

File: conda/pytorch-nightly/bld.bat [hunk 2]
```diff
@@ -112,6 +107,8 @@ IF "%USE_SCCACHE%" == "1" (
 
 if NOT "%build_with_cuda%" == "" (
     copy "%CUDA_BIN_PATH%\cudnn*64_*.dll*" %SP_DIR%\torch\lib
+    copy "%CUDA_BIN_PATH%\cudart*64_*.dll*" %SP_DIR%\torch\lib
```
Comment (malfet) [inline conda/pytorch-nightly/bld.bat:110]: Why cudart is needed?

**Review by malfet — COMMENTED** [review]
Looks fine, though why cudart embedding is needed?

File: conda/pytorch-nightly/bld.bat (hunk 2 above)
Comment (izaitsevfb) [reply conda/pytorch-nightly/bld.bat:110]: > Why cudart is needed?

To clarify a bit. There seems to be two different dll-related issues.

1. First is the missing nvtx, causing `import torch` to fail (all conda windows builds) with:
```
OSError: [WinError 126] The specified module could not be found. Error loading "C:\Users\ivanzaitsev\Anaconda3\envs\tmp_conda1\lib\site-packages\torch\lib\shm.dll" or one of its dependencies.
```

2. The second one described here (pytorch/audio#2726), only on conda windows, python 3.8 appears to be caused by missing `cudart64_110.dll`:
```
FileNotFoundError: Could not find module 'C:\Miniconda\Lib\site-packages\torchaudio\lib\libtorchaudio.pyd' (or one of its dependencies). Try using the full path with constructor syntax.
```

**Comment by izaitsevfb** [comment]
Fixes #2726

**Pull request merged**